pkg/controller/clusterversion/clusterversion_controller_test.go

```diff
@@ -89,7 +89,7 @@ func TestClusterVersionReconcile(t *testing.T) {
 				Status: configv1.ConditionTrue,
 			}),
 			validate: func(t *testing.T, cd *hivev1.ClusterDeployment) {
-				assert.Equal(t, "", cd.Annotations[constants.MinorVersionUpgradeUnavailable], "unexpected version major-minor-patch label")
+				assert.Equal(t, "", cd.Annotations[constants.MinorVersionUpgradeUnavailable], "Unexpected value for label hive.openshift.io/minor-version-upgrade-unavailable")
```

Comment:
Would have to dig into why this test was working before (as opposed to having to assert that there was no such key) but it seems accurate in its new form :shrug:

Reply:
The cd.Annotations map gets set to a non-nil value in the functional part of the code. In Go, trying to pull a map entry by a key that doesn't exist will return the 0 value of that map's value type. For example:
```
myMap := map[string]string{}
foo := myMap["foo"]
```
in this case, `foo` will be an empty string.